Sources/SwifterSwift/UIKit/UIViewExtensions.swift

```diff
@@ -751,4 +751,32 @@ public extension UIView {
     }
 }
 
+
+
+public extension UIView {
```

Comment:
Why have you added a new extension section? Can you move this into the right section? Maybe at line 703?